CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+# Contributing
+
+When contributing to this repository, please check our open issues and whether there is already an issue related to your idea. Please first discuss the change you wish to make in a GitHub issue and wait for a reply from the maintainers of this repository before making a change.
+
+We have a [code of conduct](CODE_OF_CONDUCT.md); please follow it in all your interactions relating to the project.
+
+## Rules
+
+There are a few basic ground-rules for contributors:
+
+1. **All modifications** must be made in a **pull-request** to solicit feedback from other contributors.
+2. Contributors should attempt to adhere to the prevailing [code-style](#code-style).
+
+## Pull requests
+
+**For a pull request to be merged it must at least:**
+
+:white_check_mark: &nbsp; Pass CI
+
+:white_check_mark: &nbsp; Have one approving review
+
+:white_check_mark: &nbsp; Have the PR title follow [conventional commit](https://www.conventionalcommits.org/)
+
+**Ideally, a good pull request should:**
```

Comment:
The items in this list should either consistently use or consistently not use periods

Reply:
agreed